pyproject.toml

```diff
@@ -67,6 +68,29 @@ exclude = [
 ]
 ignore-init-module-imports = true
 
+[tool.ruff.per-file-ignores]
+".actions/*" = ["S101", "S310"]
+"setup.py" = ["S101"]
+"examples/**" = [
+    "S101",
+    "S113",  # todo: Probable use of requests call without timeout
+]
```

Comment:
I recommend ignoring all codes

Reply:
Do you mean to exclude examples from code quality checks?

Reply:
I mean not forcing a particular style or special annotations like `#noqa` into example code.